src/server/routes/iframeEmbedRoute.tsx

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright (c) 2017-present, NDLA.
```

Comment:
Sjølv om den er kopiert
```suggestion
 * Copyright (c) 2023-present, NDLA.
```